flask_restx/reqparse.py

```diff
@@ -138,22 +138,24 @@ def source(self, request):
         :param request: The flask request object to parse arguments from
         """
         if isinstance(self.location, six.string_types):
-            value = getattr(request, self.location, MultiDict())
-            if callable(value):
-                value = value()
-            if value is not None:
-                return value
+            _args_location = [self.location]
         else:
-            values = MultiDict()
-            for l in self.location:
+            _args_location = self.location
+
+        values = MultiDict()
+        for l in _args_location:
+            if l == 'json' and hasattr(request, 'get_json'):
```

Comment:
Normally, `request.json` is attempted by default (property calling method `get_json`) but method's old behavior is now similar to having to pass `silent=True` parameter to the current.

Long story short: newer werkzeug package raises errors when previously it would not.

Reply:
Undid these changes and updated tests to include proper "Content-Type" header when using json body.